src/main/java/cheese/Cheese.java

```diff
@@ -0,0 +1,66 @@
+package cheese;
+
+import cheese.command.ByeCommand;
+import cheese.command.Command;
+import cheese.data.TaskList;
+import cheese.exception.CheeseException;
+import cheese.parser.Parser;
+import cheese.storage.Storage;
+import cheese.ui.Ui;
+
+/**
+ * Represents a command line interface used to manage tasks.
+ */
+public class Cheese {
+    /** List of tasks. */
+    private TaskList taskList;
+
+    /** Storage to interact with save file. */
+    private Storage storage;
+
+    /** User interface to interact with user. */
+    private Ui ui;
+
+    /**
+     * Constructs an instance of <code>Cheese</code>.
+     * 
+     * @param filePath File path of save file.
+     */
+    public Cheese(String filePath) {
+        ui = new Ui();
+        storage = new Storage(filePath);
+        try {
+            taskList = storage.load();
+        } catch (CheeseException e) {
+            ui.showError(e.getMessage());
```

Comment:
Although we're only supposed to comment on styling issues, maybe an alternative I'd like to suggest that is seems like you could probably use `Ui` as a static class, that way you don't need to keep parsing it into different parts of the code as an argument!